scripts/display.ts

```
@@ -191,32 +1036,63 @@ async function main() {
+            const typoCheck = isCloseTypo(command);
+            if (typoCheck.isTypo) {
+                const confirmed = await askDidYouMean(command);
+                if (confirmed) {
+                    console.log(c('green', '\n✓ Assuming you meant "display"...'));
+                    await showInteractiveMenu();
+                    return;
+                } else {
+                    console.log(c('red', '\n✗ Command not recognized.'));
+                    showHelp();
+                    return;
+                }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)